build_docs.pl

```diff
@@ -710,7 +710,7 @@ sub serve_and_open_browser {
         if ( not $running_in_standard_docker ) {
             sleep 1;
             say "Press Ctrl-C to exit the web server";
-            open_browser("http://localhost:8000/$open_path");
+            open_browser("http://localhost:8000/");
```

Comment:
This is always the path when you build the docs with `./build_docs.pl` which doesn't support `--all`.